tests/server/handlers/test_query_params.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from mfr.core.exceptions import ProviderError
+
+from tornado.httpclient import HTTPError
+from tests import utils
+from tornado import testing
+
+
+
+class TestRenderHandler(utils.HandlerTestCase):
+
+    @testing.gen_test
+    def test_format_url(self):
+
+        with pytest.raises(HTTPError) as e:
+            yield self.http_client.fetch(self.get_url('/export'), method='GET')
+        assert e.value.message == 'Bad Request'
+        assert e.value.code == 400
```

Comment:
Consider using `http.HTTPStatus` ([link for 3.5](https://docs.python.org/3.5/library/http.html#http.HTTPStatus)).

```python
from http import HTTPStatus
...
    assert e.value.code == HTTPStatus.BAD_REQUEST
...
```